### DISCUSSION TITLE
Chitubox shows invalid ExposureTime=1.7s (float) for sl1->uvtools->ctb

### DISCUSSION THREAD

**Discussion by msspad** [sn4k3/UVtools#160] [Q&A] [answered]
When converting a sl1 file with exposure time 1.7s in PrusaSlicer, open sl1 in UVTools and then open the automatically generated ctb in Chitubox 1.8.1, Chitubox shows 6837158 for exposure time. The property LayerExposureSeconds in UVTools shows correctly 1.7. When slicing with Chitubox, floating numbers are just fine for exposure time. Any idea why Chitubox shows 6837158 instead of 1.7s?

As a suggestion, how about adding units to the custom values in PrusaSlicer notes, e.g.  LightOffDelay_s_0, LiftHeight_mm_4 ?

btw, I'm using UVTools since it wasn't even named UVTools and it's really, really great :-)

**Comment by sn4k3** [comment, marked as answer]
Can you send me the sample file?

the note values are always on same units, s and mm/m. Changing the names now are bad for compability

**Reply by msspad** [reply]
just made a simple sample file and found the issue: While the UI field for exposure time in Chitubox shows e.g. 6837158, you can activate the UI field, go to the left and see that's in fact 1.7000000476837158 . So it's rather a numeric issue combined with an UI issue in Chitubox. I guess the ctb works fine with the printer (will test tomorrow). In case you want to see it anyway, I attached [lid.zip](https://github.com/sn4k3/UVtools/files/6057313/lid.zip) (only zip allowed on github).

**Reply by sn4k3** [reply]
Yes, chitubox don't round floats, but i guess on firmware they do. UVtools by default are rounding those to 2

**Reply by msspad** [reply]
as expected, it printed just fine, thanks anyway
